tests/utils/injectStylesheet.test.ts

```diff
@@ -68,4 +73,66 @@ describe( 'injectStylesheet', () => {
 			`Stylesheet with "${ CDN_MOCK_STYLESHEET_URL }" href is already present in DOM!`
 		);
 	} );
+
+	it( 'should inject the stylesheet at the end of the head if headPlacement = \'end\'', async () => {
```

Comment:
That's off-topic. I know we require single quotes. But in such cases, doesn't this seem cleaner? Or it's just me?
```suggestion
	it( "should inject the stylesheet at the end of the head if headPlacement = 'end'", async () => {
```

Reply:
I got you. I was thinking more about updating these rules. But that's off-topic.